Cabal-syntax/src/Distribution/Backpack.hs

```diff
@@ -1,51 +1,50 @@
-{-# LANGUAGE DeriveDataTypeable         #-}
-{-# LANGUAGE DeriveGeneric              #-}
-{-# LANGUAGE FlexibleInstances          #-}
+{-# LANGUAGE DeriveDataTypeable #-}
```

Comment:
It's pretty funny that it doesn't like aligning braces. I wonder if that's configurable. 

I have no hard opinion on this but I kinda liked the aligning. Of course, it's painful to modify...

Reply:
It's actually part of the diff reducing. You don't want that kind of extra noise when adding a new pragma during the review. :)

Reply:
I mean I kinda don't but I'd be willing to trade this for pleasant reading. And i read code much more often than doing reviews.

Anyways, curious what others think.

Reply:
Aligned braces are pretty and suck to maintain. It's not more pleasant imo - it just satisfies a particular "oo this aligns" feeling that doesn't actually contribute to legibility.